daemon/attach.go

```diff
@@ -60,6 +60,7 @@ func (daemon *Daemon) ContainerAttach(prefixOrName string, c *backend.ContainerA
 
 	if err := daemon.containerAttach(container, stdin, stdout, stderr, c.Logs, c.Stream, keys); err != nil {
 		fmt.Fprintf(outStream, "Error attaching: %s\n", err)
+		return err
```

Comment:
Hi, @coolljt0725 , Thanks for your feedback. Currently I am not so familiar with the stream thing. While I just feel that if `containerAttach` fails, the client can only sense a error log? and no non-zero exit code returns?
I think I need some time to understand this part.
And feel free to close this if it is improper. :)